bin/MadGraph5_aMCatNLO/cards/ggh01_Toa01a01_Tomumutautau/haa_h125_10_gp/ggh01_M125_Toa01a01_M10_Tomumutautau_customizecards.dat

```diff
@@ -0,0 +1,3 @@
+set param_card mass 25 125
+set param_card mass 36 10
```

Comment:
is the default width in sync with your modified mass settings? shouldn't you update?

Reply:
We do not intend to change the decay widths of "h" and "a" to complement our modified mass settings. In the mass range we are interested in, the decay width of "a" is tiny and the ratio of width/mass for "h" is really small so we think it is reasonable to ignore any variation in decay widths. Given the limited resolution of the detector in reconstructing the energy of the particles, we are bound to see a smearing effect on our mass distributions. The widths of the generated mass distributions are negligible compared to the smeared spectrum.

Reply:
Just curious, what are the default decay widths?

Reply:
DECAY 25 3.037863e-02 #  wh01
DECAY 36 2.495587e-04 #  wa01